sdk/python/tests/integration/registration/test_inference.py

```diff
@@ -138,6 +140,29 @@ def test_update_data_sources_with_inferred_event_timestamp_col(simple_dataset_1)
             )
 
 
+@pytest.mark.integration
+@pytest.mark.universal
+def test_update_data_sources_with_inferred_event_timestamp_col(universal_data_sources):
+    (_, _, data_sources) = universal_data_sources
+    data_sources_copy = deepcopy(data_sources)
+
+    # remove defined event_timestamp_column to allow for inference
+    for data_source in data_sources_copy.values():
+        data_source.event_timestamp_column = None
+
+    update_data_sources_with_inferred_event_timestamp_col(
+        data_sources_copy.values(),
```

Comment:
not sure why your local linter isn't working, but what i see locally is this should be all in a single line:

```
data_sources_copy.values(), RepoConfig(provider="local", project="test"),
```